registrations/models.py

```diff
@@ -67,40 +67,53 @@ def formfield(self, **kwargs):
         )
 
 
-class Registration(models.Model):
-    event = models.OneToOneField(
-        Event,
-        on_delete=models.CASCADE,
-        related_name="registration",
-        null=False,
+class CreatedModifiedBaseModel(models.Model):
+    created_at = models.DateTimeField(
+        verbose_name=_("Created at"),
+        null=True,
```

Comment:
Is there a purpose for nullable auto_now_add field?

Reply:
In this case it's because the SignUp model has existed before these changes and there can be already created instances in the database. For those it is probably most accurate to have a null value and due to this, we allow this field to be nullable.

Reply:
This field has now been made non-nullable.